gto/git_utils.py

```diff
@@ -281,9 +294,13 @@ def git_add_and_commit_all_changes(repo_path: str, message: str) -> None:
         repo.index.commit(message=message)
 
 
+def read_repo(repo: Union[str, git.Repo]) -> git.Repo:
+    return git.Repo(path=repo) if isinstance(repo, str) else repo
+
+
 @contextmanager
 def stashed_changes(repo_path: str, include_untracked: bool = False):
```

Comment:
need to update the signature of this function then, `repo_path: str` should probably change into `repo: Union[str, git.Repo]`?